values/tenant-values.yaml

```
@@ -6,6 +6,10 @@ tenant:
   configuration:
     name: ###ZARF_VAR_TENANT###
     existingSecret: true
+  configSecret:
```

```suggestion
  configSecret:
```
Looking through the templates that should be in configSecret like you have it (`configuration.existingSecret` is not used)